### ISSUE TITLE
When Ogarul goes down Whats the Situation?

### ISSUE THREAD

**Issue by ZlkGamingYT** [AJS-development/Ogar-unlimited#1453]
Please read the FAQ in advance.

YOU MUST SEARCH UP YOUR ERROR FIRST TO SEE IF IT IS REPRODUCED

What game mode were you in when you had the issue?

What version are you on?

What command did you type if any?

You must have a description as well

Most common questions involve:

Why is there an error "Cannot find module"
A: Do npm install after you cd into the repo.

VIOLATORS OF THIS WILL BE BANNED FOR 1 MONTH

**Comment by ZlkGamingYT** [comment]
When Ogarul goes Down (which it obviously will) will people be able to copy the sites info to make a fake?? Also is there going to be a site/new development? Cuz ogarul is definitley popular enough to stay alive!

**Comment by ZlkGamingYT** [comment]
What i meant was is the copyright sysytem going to be broken to allow many sites that could have malicious intent//Be the real version?

**Comment by rydens** [comment]
I'm pretty sure the repo will stay here, and according to the creator, all support will be gone, so no "issues".

Also, as for the new one, I have no idea how they are planning to make that, but there won't be as much support for that as they had here.

I don't know about the "copyright system"... sorry

**Comment by rydens** [comment]
You can try reading the full license for this software, found [here](http://www.apache.org/licenses/LICENSE-2.0).

**Comment by ZlkGamingYT** [comment]
I just know that TryDox got sued for faking...smh and thanks for the info.

**Comment by ZlkGamingYT** [comment]
Thanks

**Comment by SharkFinPro** [comment]
Im hosting the site

**Comment by ZlkGamingYT** [comment]
Ok cool what will the site be called?

**Comment by LegitSoulja** [comment]
See with trydox, he literally took everything, even files that wasn't even part of the client. That's why I had a little talk with him.

But now the site is down completely since I talked to my lawyer about the situation. And btw, the site isn't an Apache license, it's my own license. My so what company/group.

I won't release the entire client because there's a lot of things I'll have to go in and remove before I make it public, but I'm not doing that. I provided the skins. That's all that I can do as of right now.
